tests/test_basic.py

```diff
@@ -1,5 +1,46 @@
 import json
 import os
+import jwt
+import time
+import pytest
+
+secret_key = 'secretkey_test'
+default_exp_time = int(time.time()) + 5000
+default_token_payload = dict(
+    sub="mtm@mtmco.net",
+    name="mmeharga",
+    roles="general",
+    exp=default_exp_time,
+    tem=0,
+    mstout=True,
+    mssub=True
+)
+
+
+@pytest.mark.slow
+def test_show_product(dispatcher_live_fixture, capsys):
+    with capsys.disabled():
+        from oda_api.api import DispatcherAPI
+
+        disp = DispatcherAPI(url=dispatcher_live_fixture, instrument='mock')
+        # let's generate a valid token
+        token_payload = {
+            **default_token_payload,
+            "roles": ["general", "integral-private-qla"],
+        }
+        encoded_token = jwt.encode(token_payload, secret_key, algorithm='HS256')
+        products = disp.get_product(instrument="isgri",
+                                    product="isgri_lc",
+                                    product_type="Dummy",
+                                    osa_version='OSA11.0',
+                                    token=encoded_token
+                                    )
+
+    products.show()
+
+    captured = capsys.readouterr()
+    print("Show() output: ", captured.out)
+    assert captured.out == "ID=0 prod_name=isgri_lc_0_OAO1657m415  meta_data: {'src_name': 'OAO 1657-415', 'time_bin': 0.0115739687598762, 'time': 'TIME', 'rate': 'RATE', 'rate_err': 'ERROR'}\n\n"
```

Comment:
Can we also here check that the result is available as `products.isgri_lc_0_OAO1657m415` and also in products._n_list and products._p_list